### ISSUE TITLE
Database

### ISSUE THREAD

**Issue by tinyhostingltd** [peteog/Closebox#1]
Nice app but where does the database come from?

Seems inaccurate to me, missing postboxes and incorrect times.

**Comment by peteog** [comment]
I agree, it's a tricky dataset to keep correct and up to date. The dataset originally comes from the Royal Mail (https://www.whatdotheyknow.com/request/location_of_every_post_box_that) combined with OpenStreetMap data (http://wiki.openstreetmap.org/wiki/Post_box). The guy that runs http://dracos.co.uk has a CSV feed that he keeps fairly up to date.

I originally considered having a cloud based database where users could submit additions and corrections, but I never had time to implement it.
